options/options.c

```diff
@@ -216,6 +219,9 @@ const struct m_sub_options vo_sub_opts = {
         .hidpi_window_scale = true,
         .native_fs = true,
         .taskbar_progress = true,
+#if HAVE_WIN32_DESKTOP
+        .window_affinity = WDA_NONE,
+#endif
```

Comment:
Since `WDA_NONE` is just zero, you don't need to set this because mpv already initializes the option as 0.

Reply:
Can I just fully omit this line?

Reply:
Yeah you can get rid of all of these lines.

Reply:
Done, also applied all the other changes and added docs entry